crates/bevy_pbr/src/material.rs

```diff
@@ -795,6 +796,24 @@ pub struct PreparedMaterial<T: Material> {
 #[derive(Component, Clone, Copy, Default, PartialEq, Eq, Deref, DerefMut)]
 pub struct MaterialBindGroupId(Option<BindGroupId>);
 
+#[derive(Default)]
+pub struct AtomicMaterialBindGroupId(AtomicU32);
+
+impl AtomicMaterialBindGroupId {
+    pub fn set(&self, id: MaterialBindGroupId) {
+        let id = if let Some(id) = id.0 {
+            core::num::NonZeroU32::from(id).get()
```

Comment:
Can just be `NonZeroU32` given that you imported it already

Reply:
Done.